blog/2025/integrating-dia-oracles-on-xrpl.md

```diff
@@ -29,13 +29,13 @@ The providers to fetch data from:
 
 Each oracle document contains real-time pricing data for key assets—including XRP, RLUSD, and more — making it simple for developers to retrieve trusted market prices directly from the ledger.
 
-### 1.2 Account Funding for Transaction Fees
+### 1.2 Oracle Accounts
 
-To ensure uninterrupted oracle operation when pushing data to your contracts, it is crucial to maintain a sufficient XRP balance in the relevant XRPL account. This balance must cover the XRP reserve requirements for the account and the minor transaction fees incurred with each transaction. You can monitor the accounts listed below to ensure they remain adequately funded at all times.
+DIA uses the following accounts to publish oracle price data to the XRPL:
```

Comment:
Confirming that the note about maintaining sufficient account balance no longer needed.